packages/slate-react/src/components/editable.tsx

```diff
@@ -624,19 +624,52 @@ export const Editable = (props: EditableProps) => {
               ) {
                 const node = ReactEditor.toSlateNode(editor, event.target)
                 const path = ReactEditor.findPath(editor, node)
-                const start = Editor.start(editor, path)
-                const end = Editor.end(editor, path)
+                // Check if user clicks thrice
+                if (event.detail === 3) {
```

Comment:
Have you thought about setting this to `event.detail >= 3` instead? I noticed when trying a similar approach to this fix locally that if you spam click many times, `event.detail` continues to increment so this wouldn't be fired once you go past 3 clicks.